tests/suite/test_prometheus_metrics.py

```diff
@@ -173,3 +186,48 @@ def test_latency_metrics(
         for item in expected_metrics:
             assert item in resp_content
 
+    @pytest.mark.parametrize(
+        "ingress_controller, expected_metrics",
+        [
+            pytest.param(
+                {"extra_args": ["-enable-prometheus-metrics", "-enable-latency-metrics", "-prometheus-tls-secret=nginx-ingress/prometheus-test-secret"]},
+                [
+                    'nginx_ingress_controller_ingress_resources_total{class="nginx",type="master"} 0',
+                    'nginx_ingress_controller_ingress_resources_total{class="nginx",type="minion"} 0',
+                ],
+            )
+        ],
+        indirect=["ingress_controller"],
+    )
+    @pytest.mark.sean
```

Comment:
I'm guessing this was left in by accident?

Reply:
fixed and added a local githook to prevent further issues.